_data/_samples.json

```diff
@@ -312,6 +312,7 @@
       "tags": [
         "Beginner",
         "SPI",

```

Comment:
"title": "SPI accelerometer" is misleading - might be better to come up with a different title for the users searching for I2C samples.
What is the difference between the "I2C accelerometer" sample and the "SPI accelerometer" sample configured to use I2C? Do we have duplication here or there is a difference we can call out?